### PR TITLE
Insert more tests in test analyzer

### PR DESCRIPTION
**Pull request by GiasemiSh**
Now we can provide the test we want to run through the configuration.

### CODE REVIEW

**Comment by GiasemiSh** [comment]
Test it with: https://github.com/QMSTR/qmstr-demo/pull/34

File: cmd/analyzers/test-analyzer/main.go [hunk 1]
```diff
@@ -23,9 +29,34 @@ func main() {
 		log.Printf("%v failed: %v\n", analyzer.GetModuleName(), err)
 		os.Exit(master.ReturnAnalyzerFailed)
 	}
+	for _, test := range tests {
```
Comment (hemarkus) [inline cmd/analyzers/test-analyzer/main.go]: Why do you call the tests here? This seems plain wrong.

Comment (GiasemiSh) [reply cmd/analyzers/test-analyzer/main.go]: I want to test against Package node. Analyze function is called for each file node. 
Then I should implement another function for that right?

Comment (hemarkus) [reply cmd/analyzers/test-analyzer/main.go]: Right. Please create an issue for this. Then we should merge this.

**Pull request merged**